packages/building-utils/index-html/extract-resources.js

```diff
@@ -21,7 +23,11 @@ function extractResources(htmlString) {
 
   const scripts = queryAll(indexHTML, predicates.hasTagName('script'));
   const moduleScripts = scripts.filter(script => getAttribute(script, 'type') === 'module');
+  const importMapScripts = scripts.filter(script => getAttribute(script, 'type') === 'importmap');
+
   const jsModules = [];
+  const importMaps = [];
+  const importMapPaths = [];
```

Comment:
I think we need to be a bit clearer around the naming conventions. Maybe `inlineImportMaps` and `importMapPaths` ? And similarly we will need to rename `jsModules` to `inlineModules` and `modulePaths` but we can do that later.

What do you think?

Reply:
sounds like a good idea 🤗

Reply:
changed